addons/l10n_ee/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,350 @@
+                                        <field name="figure_type">integer</field>
+                                    </record>
+                                </field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="tax_report_line_5_4" model="account.report.line">
+                        <field name="name">5.4 - VAT paid or payable on acquisition of a car used partially for business purposes, and on acquisition of goods and receipt of services for such car</field>
+                        <field name="code">l10n_ee_vat_5_4</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_line_5_4_tag" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">5_4</field>
+                            </record>
+                        </field>
+                        <field name="children_ids">
+                            <record id="tax_report_line_5_4_cars" model="account.report.line">
+                                <field name="name">Number of cars</field>
+                                <field name="expression_ids">
+                                    <record id="tax_report_line_5_4_cars_value" model="account.report.expression">
+                                        <field name="label">balance</field>
+                                        <field name="engine">external</field>
+                                        <field name="formula">most_recent</field>
+                                        <field name="subformula">editable;rounding=0</field>
+                                        <field name="figure_type">integer</field>
+                                    </record>
+                                </field>
+                            </record>
+                        </field>
+                    </record>
+                </field>
+            </record>
+            <record id="tax_report_line_6" model="account.report.line">
+                <field name="name">6 - Intra-Community acquisitions of goods and services received from a taxable person of another Member State, total, incl.</field>
+                <field name="code">l10n_ee_vat_6</field>
+                <field name="expression_ids">
+                    <record id="tax_report_line_6_tag" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">6</field>
+                    </record>
+                </field>
+                <field name="children_ids">
+                    <record id="tax_report_line_6_1" model="account.report.line">
```

Comment:
@dylankiss It is not normal to have to put a tax into 2 tax grids where one is a child of the other.  
We can solve this easily by putting 2 expression_ids in the line above: 
- One with the aggregation as label balance (6.1 + 6)
- One to calculate the tax_tags with only 6 (which will not show)

It is not only for the 6/6.1, but also 3/3.1/3.1.1, ...